.travis.yml

```diff
@@ -3,45 +3,48 @@ sudo: required
 dist: trusty
 
 env:
- - PYTEST_QT_API=pyqt4   PYTHON=python2.7
- - PYTEST_QT_API=pyqt4v2 PYTHON=python2.7
- - PYTEST_QT_API=pyside  PYTHON=python2.7
+ - PYTEST_QT_API=pyqt4   PYTHON_VERSION=2.7
+ - PYTEST_QT_API=pyqt4v2 PYTHON_VERSION=2.7
+ - PYTEST_QT_API=pyside  PYTHON_VERSION=2.7
 
- - PYTEST_QT_API=pyqt4   PYTHON=python3.4
- - PYTEST_QT_API=pyqt4v2 PYTHON=python3.4
- - PYTEST_QT_API=pyside  PYTHON=python3.4
- - PYTEST_QT_API=pyqt5   PYTHON=python3.4
+ - PYTEST_QT_API=pyqt4   PYTHON_VERSION=3.4
+ - PYTEST_QT_API=pyqt4v2 PYTHON_VERSION=3.4
+ - PYTEST_QT_API=pyside  PYTHON_VERSION=3.4
+ - PYTEST_QT_API=pyqt5   PYTHON_VERSION=3.4
 
 install:
  - sudo apt-get update
 
  # Xvfb / window manager
  - sudo apt-get install -y xvfb herbstluftwm
 
- # Qt
- - $PYTHON scripts/install-qt.py
-
- # Pip (3.4 does not have it by default)
- - wget https://bootstrap.pypa.io/get-pip.py
- - sudo $PYTHON get-pip.py
+ # Install 
+ - git clone --depth 1 git://github.com/astropy/ci-helpers.git
+ - export SETUP_XVFB=true && source ci-helpers/travis/setup_conda.sh
 
  # Dependencies
- - sudo $PYTHON -m pip install -U pytest tox coveralls
+ - |
+     if [[ ${PYTEST_QT_API} == 'pyside' ]]; then
+         pyqt_package="pyside==1.2.4"
```

Comment:
What do you think about creating a `PYQT_PACKAGE` environment variable in the `env` section along with the others? There will be a small repetition, but I think that's OK.